packages/wonder-blocks-testing/src/fetch/mock-fetch.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import {fetchRequestMatchesMock} from "./fetch-request-matches-mock.js";
+import {mockRequester} from "../mock-requester.js";
+import type {FetchMockFn, FetchMockOperation} from "./types.js";
+
+/**
+ * A mock for the fetch function passed to GqlRouter.
+ */
+export const mockFetch = (): FetchMockFn =>
+    mockRequester<FetchMockOperation, _>(
```

Comment:
Thanks to making the `mockGqlFetch` implementation more generic, we only need to provide the matcher and a function to turn a request into helpful text.